From eae1ca0a0a23e245742cfb32743384d1443a3ae1 Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai.pt@gmail.com>
Date: Mon, 26 Dec 2022 12:04:32 +0800
Subject: [PATCH linux dev-6.0 1/2] ARM: dts: aspeed: bletchley: rename flash1 label

In OpenBMC, phosphor-software-manager use "alt-bmc" for secondary falsh
label.
Rename flash1 label to "alt-bmc" to support dual image feature inOpenBMC.

Signed-off-by: Potin Lai <potin.lai.pt@gmail.com>
---
 arch/arm/boot/dts/aspeed-bmc-facebook-bletchley.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/aspeed-bmc-facebook-bletchley.dts b/arch/arm/boot/dts/aspeed-bmc-facebook-bletchley.dts
index a619eec70633..791f83aaac50 100644
--- a/arch/arm/boot/dts/aspeed-bmc-facebook-bletchley.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-facebook-bletchley.dts
@@ -307,7 +307,7 @@ flash@0 {
 	flash@1 {
 		status = "okay";
 		m25p,fast-read;
-		label = "flash1";
+		label = "alt-bmc";
 		spi-max-frequency = <50000000>;
 	};
 };
-- 
2.31.1

